pkg/gardenlet/controller/controllerinstallation/controllerinstallation_control.go

```diff
@@ -216,14 +216,25 @@ func (c *defaultControllerInstallationControl) reconcile(controllerInstallation
 		}
 	}
 
+	gardenClusterIdentity, err := getClusterIdentity(ctx, gardenClient.Client(), nil)
```

Comment:
garden cluster identity is known to the gardenlet. it is part of its initialization, and we can skip this get.

Reply:
During the seed cluster bootstrap, if the seed doesn't have the cluster-identity in its status ( the status.clusterIdentity is immutable ), we compute it and then create the config map, so I am wondering if we should even try to get the config map or directly return an error.

Reply:
thanks for the hints, I have adjusted it accordingly.